classes/image_populate_mfgtool.bbclass

```diff
@@ -0,0 +1,142 @@
+# Allow generation of mfgtool bundle
+#
+# The class provides the infrastructure for MFGTOOL generation and is tied to images. To generate
+# the bundle, the task populate_mfgtool must be called. For exemple:
+#
+# ,----[ Running populate_mfgtool for core-image-minimal image ]
+# | $: bitbake core-image-minimal -c populate_mfgtool
+# `----
+#
+# The class behavior is controlled through the MFGTOOLSCRIPT variable. The MFGTOOLSCRIPT variable
+# itself specifies a space-separated list of the scripts to enable. Following the scripts, you can
+# determine the behavior of each script by providing up to tow order-dependent arguments, which are
+# separated by commas. You can omit any argument you like but must retain the separating commas. The
```

Comment:
the arguments are separated by commas. Maybe the list (for each argument) is separated by spaces?

Reply:
The use of `,` is important as we can have multi dependencies and files.

Reply:
I think I improved it.